test/App/Fossa/ManualDepsSpec.hs

```diff
@@ -53,8 +53,9 @@ theWorks = ManualDependencies references customs vendors remotes locators
       , RemoteDependency "url-dep-two" "1.2.4" "www.url2.tar.gz" Nothing
       ]
     vendors =
-      [ VendoredDependency "vendored" "path" Nothing
-      , VendoredDependency "versioned" "path/to/dep" (Just "2.1.0")
+      [ VendoredDependency "vendored" "path" Nothing Nothing
+      , VendoredDependency "versioned" "path/to/dep" (Just "2.1.0") Nothing
+      , VendoredDependency "metadata" "path" (Just "1.1.0") (Just (DependencyMetadata (Just "description for vendored") (Just "we don't validate homepages - vendored")))
```

Comment:
Should we validate these to ensure that they are URLs? If they send an invalid one up to Core would it just put invalid data in the UI/report? I guess more generally, where would we rather the failure happen in this case?

Reply:
Good question - as far as I know, we only display the homepage in the UI / reports, I'm not too concerned about invalid URLs being reported but it might warrant a discussion with the team.

Reply:
I think it's reasonable as-is if that's the case. On second thought thinking about my initial question I think it's hopefully obvious to the customer that it's their fault if one of these urls is messed up.